library/std/src/fs.rs

```diff
@@ -1197,7 +1197,7 @@ impl OpenOptions {
 
     /// Sets the option for truncating a previous file.
     ///
-    /// If a file is successfully opened with this option set it will truncate
+    /// If a file is successfully opened with this option set as true, it will truncate
```

Comment:
```suggestion
    /// If a file is successfully opened with this option set to true, it will truncate
```
I think to reads better, you set an option to something

Reply:
I would agree.